frmts/nitf/rpftocfile.cpp

```diff
@@ -549,17 +549,14 @@ RPFToc* RPFTOCReadFromBuffer(const char* pszFilename, VSILFILE* fp, const char*
         frameEntry->filename[12] = '\0';
         bOK &= strlen(frameEntry->filename) > 0;
 
-        /* Check if the filename is an overview or legend */
-        for( int j = 0; j < 12; j++ )
-        {
-            if (strcmp(&(frameEntry->filename[j]),".OVR") == 0 ||
-                strcmp(&(frameEntry->filename[j]),".ovr") == 0 ||
-                strcmp(&(frameEntry->filename[j]),".LGD") == 0 ||
-                strcmp(&(frameEntry->filename[j]),".lgd") == 0)
+        // Check (case insensitiv) if the filename is an overview or legend
+        // some CADRG maps have legend name smaller than 8.3 then the extension
+        // has blanks (0x20) at the end -> check only the first 3 letters of the extension.
+        const char* fileExt = CPLGetExtension(frameEntry->filename);
+        if (EQUALN(fileExt, "ovr", 3) ||
+            EQUALN(fileExt, "lgd", 3))
```

Comment:
the indentation of { at next line is not consistent with the matching }

Reply:
right - looks like a merge error. I will fix it.